### PR TITLE
fix(CDAP-20647): Improve SCM config not found error message and add direction where to configure.

### PR DESCRIPTION
**Pull request by de-lan**
## WHAT
Improve SCM config not found error message and add direction where to configure.

Sister UI PR: https://github.com/cdapio/cdap-ui/pull/1054

### CODE REVIEW

**Review by sumengwang — APPROVED** [review]

**Review by tivv — COMMENTED** [review]
Please add a test. Also I believe the `NamespaceIs.toString` would be something like `namespace:aaa` that is redundant for the message. Test should show it

File: cdap-common/src/test/java/io/cdap/cdap/common/RepositoryNotFoundExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+/** Tests fpr RepositoryNotFoundException */
+public class RepositoryNotFoundExceptionTest {
+  @Test
+  public void testRepositoryNotFoundExceptionMessage() {
+    RepositoryNotFoundException exception = new RepositoryNotFoundException(NamespaceId.DEFAULT);
+
+    Assert.assertEquals(
+        exception.getMessage(),
+        String.format("There is no repository configuration for namespace %s. "
+            + "Please configure it in Namespace Admin page.", NamespaceId.DEFAULT.getNamespace()));
```
Comment (tivv) [inline cdap-common/src/test/java/io/cdap/cdap/common/RepositoryNotFoundExceptionTest.java]: Please don't use String.format over here. It does not give much, but prevents from seeing exact error message served to user

Comment (tivv) [reply cdap-common/src/test/java/io/cdap/cdap/common/RepositoryNotFoundExceptionTest.java]: Also I believe in `org.junit.Assert#assertEquals(java.lang.Object, java.lang.Object)` expected goes first in params

Comment (de-lan) [reply cdap-common/src/test/java/io/cdap/cdap/common/RepositoryNotFoundExceptionTest.java]: updated

**Review by tivv — APPROVED** [review]

**Pull request merged**